addons/im_livechat/static/tests/tours/im_livechat_session_agent_display_name_tour.js

```diff
@@ -0,0 +1,18 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("im_livechat_session_agent_display_name_tour", {
+    steps: () => [
+        {
+            trigger: ".o_searchview_dropdown_toggler",
+            run: "click",
+        },
+        {
+            trigger: ".o_group_by_menu .dropdown-item:contains('Agent')",
+            run: "click",
+        },
+        {
+            trigger:
+                ".o_kanban_group:not(.o_column_folded) .o_column_title:not(:has(.text-truncate:contains('TestCompany')))",
```

Comment:
This test is very weak. Negative assertion on a complex selector. I would just add a test for display name on the python side to ensure company is properly omitted.